repo/packages/P/percona-pxc-mysql/100/resource.json

```diff
@@ -0,0 +1,70 @@
+{
+  "assets": {
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u162-linux-x64.tar.gz",
+      "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.11.0.tar.gz",
+      "bootstrap-zip": "http://downloads.mesosphere.com/dcos-commons/artifacts/0.42.1/bootstrap.zip",
+      "scheduler-zip": "https://pxc-bucket.s3.amazonaws.com/pxc/assets/0.2.0-5.7.21/percona-pxc-mysql-scheduler.zip",
+      "executor-zip": "http://downloads.mesosphere.com/dcos-commons/artifacts/0.42.1/executor.zip",
+      "pip-uri": "https://bootstrap.pypa.io/get-pip.py",
+      "zk-uri": "https://s3-ap-northeast-1.amazonaws.com/pxc-utils/zookeepercli-linux-amd64-binary.tar.gz",
+      "mysqld-exporter-uri": "https://s3-ap-northeast-1.amazonaws.com/pxc-utils/mysqld_exporter-0.7.1.linux-amd64.tar.gz",
+      "mc-uri": "https://dl.minio.io/client/mc/release/linux-amd64/mc"
+    },
+    "container": {
+      "docker": {
+        "percona-xtradb-cluster-root": "mesostcs/pxc:5.7-DCOS-root",
+        "percona-xtradb-cluster-nobody": "mesostcs/pxc:5.7-DCOS-nobody",
+        "proxysql-root": "mesostcs/proxysql:1.4.12-DCOS-root",
+        "proxysql-nobody": "mesostcs/proxysql:1.4.12-DCOS-nobody",
+        "mysqlcli": "mysql:5.7",
+        "prom-mysqld-exporter": "prom/mysqld-exporter:0.11.0"
```

Comment:
@ishanchandrajoshi one of the checks is failing because this image does not exits. Should this not be on mesostcs.

cc @ryadav88